### ISSUE TITLE
Restart Pecan on Emu with Geoduck files

### ISSUE THREAD

**Issue by laurahspencer** [sr320/LabDocs#526]
@seanb80 please run the following 4 sample files in the following order:

Geoduck1   (corresponds to Geoduck 48 @ Site Fidalgo Bay = Eelgrass)
Geoduck6  (corresponds to Geoduck 58 @ Site  Fidalgo Bay - Bare)
Geoduck3  (corresponds to Geoduck 10 @ Site Case Inlet - Eelgrass)
Geoduck9  (corresponds to Geoduck 18 @ Site Case Inlet - Bare)

When these are done I'll have another set to run.

**Comment by laurahspencer** [comment]
Chatted with Sean to sketch out this next Pecan run timeline.

If we use 4 of the 16 logical cores on Emu (we used 3 in the recent test run, and tried 14 in the big run that didn't work / was cancelled), using the 80 isolation windows: it should take ~13.5 days to complete 2 data files (aka 2 samples).  

This is assuming that only using 3 isolation windows and 3 logical cores = 10 hours per sample file, and that increasing the logical cores from 3 to 4 results in a linear decrease in time/isolation window.

Bottom line, with the given settings I should have data from 1 sample file done in a week, which I'll plan on using for the poster.

**Comment by seanb80** [comment]
It looks like Pecan/Grid Engine is still using much more memory than I gave it permission to, but hopefully with fewer processes running this won't end poorly. Will monitor it over the weekend to make sure it's behaving properly. 

<img width="573" alt="screen shot 2017-03-10 at 5 48 29 pm" src="https://cloud.githubusercontent.com/assets/10619258/23819343/de12e3ce-05b9-11e7-830a-5185dd5cdfac.png">

**Comment by laurahspencer** [comment]
FYI Sean walked me through killing/restarting Pecan since the memory issue persisted over the weekend and resulted in partially-completed runs. 

See my [latest lab notebook entry](https://laurahspencer.github.io/LabNotebook/More-Pecan-Troubleshooting/)

**Comment by laurahspencer** [comment]
Update: Steven prepared a shortened background proteome with only stress-response related proteins (see issue #528) - selected 600 proteins from 30k.  I restarted Pecan - check out [my latest blog post](https://laurahspencer.github.io/LabNotebook/Pecan-Again/) for details.

**Comment by laurahspencer** [comment]
FYI the input files for my latest Pecan run are all here: http://owl.fish.washington.edu/generosa/Generosa_DNR/Pecan/
